packages/browser-service/src/browser.service.ts

```diff
@@ -119,50 +124,77 @@ export class BrowserService {
     });
 
     await this.browserContextService.initBrowserContext();
+    await this.annotateExtensionWalletVersion(extensionService);
+    this.setWalletPage();
+    await this.getEOAWalletPage().setup(this.options.networkConfig.chainName);
+  }
 
-    if (
-      this.options.walletConfig.WALLET_TYPE === WalletTypes.EOA &&
-      !!process.env.CI
-    ) {
-      const manifestContent = await extensionService.getManifestContent(
-        this.options.walletConfig.STORE_EXTENSION_ID,
-      );
-      test.info().annotations.push({
-        type: 'wallet version',
-        description: manifestContent.version,
-      });
+  async teardown() {
+    if (this.browserContextService.browserContext !== null)
+      await this.browserContextService.browserContext.close();
+    if (this.ethereumNodeService) {
+      await this.ethereumNodeService.stopNode();
+    }
+  }
+
+  private getStandUrlByNetwork(): string {
+    switch (this.options.networkConfig.chainName) {
+      case 'Ethereum Hoodi':
+        return 'https://stake-hoodi.testnet.fi';
```

Comment:
we are using browser context for another apps, like csm, dao - i think we cannot use url specified for si widget